biz.aQute.repository/src/aQute/bnd/repository/p2/provider/P2Indexer.java

```diff
@@ -51,6 +57,17 @@
 	private final File						indexFile;
 	private volatile BridgeRepository		bridge;
 	private static final Resource		RECOVERY	= new ResourceBuilder().build();
+	private static final String			P2_CAPABILITY_NAMESPACE	= "bnd.p2";
+	private static final String			MD5_ATTRIBUTE			= "md5";
+	private static final Requirement	MD5_REQUIREMENT;
+	static {
+		try {
+			MD5_REQUIREMENT = new RequirementBuilder(P2_CAPABILITY_NAMESPACE).addAttribute(MD5_ATTRIBUTE, "*")
```

Comment:
Oh okay thanks, I didn't realise it would be ignored the way I did it. I just thought it felt a bit neater than formatting as a filter string, but yes that all makes sense I'll sort it out.